features/ouders.feature

```diff
@@ -288,7 +288,7 @@ Rule: Wanneer de geslachtsnaam van de ouder onbekend is, wordt de ouder geleverd
     | naam                 | waarde        |
     | type                 | OnbekendOuder |
     | indicatieOnbekend    | true          |
-    | ouderAanduiding.code | ouder1        |
+    | ouderAanduiding.code | 1             |
```

Comment:
idem: ".code" verwijderen

Reply:
> idem: ".code" verwijderen

moet niet worden verwijderd. Er is/bestaat geen GBA element mapping hiervoor. Dit wordt toch geconcludeerd uit de categorie?

Reply:
My mistake. Is geen waardetabel meer. Klopt